akka-remote/src/test/scala/akka/remote/artery/RemoteSendConsistencySpec.scala

```diff
@@ -62,6 +61,17 @@ abstract class AbstractRemoteSendConsistencySpec(config: Config)
   val addressB = address(systemB)
   val rootB = RootActorPath(addressB)
 
+  private def actorRefBySelection(path: ActorPath) = {
+
+    val correlationId = UUID.randomUUID().toString
+    system.actorSelection(path) ! Identify(Some(correlationId))
+
+    val actorIdentity = expectMsgType[ActorIdentity](5.seconds)
+    actorIdentity.correlationId shouldBe correlationId
+
+    actorIdentity.ref.get
```

Comment:
If it turns out this fails sometimes (message lost or race condition, not obvious it could happen to me though) we could wrap all of it in a `awaitAssert { }` to do retries within some timeframe.

Reply:
If we confirm that the first `ActorIdentity` is taking so long, we should try to understand why. 

If my assumption [here](https://github.com/akka/akka/issues/27954#issuecomment-620539197) is correct, the first message was delivered, but 7 seconds later (at least). On top of the initial 5 seconds the next test consumes at least 2 more seconds.

Reply:
@johanandren, I understood that you are suggesting here a possible improvement in case this fails sometimes and not something to integrate on this PR. So I merge as is. 

Btw, the PR is not trying to fix the failure, but to give more visibility. I do think we are going to see it failing again. But with more information we can have a better fix. 

So, I will keep an eye on it.